code/modules/reagents/reagent_containers/food/cans.dm

```diff
@@ -178,28 +178,28 @@
 
 /obj/item/reagent_containers/food/drinks/cans/monster_red
 	name = "\"Code RED\" Claw Energy"
-	desc = "Claw energy, for when you need the focus of a hawk and the energy of a cheetah. A warning label says drinking to much may cause heart damage, liver damage, and not living anymore. \
+	desc = "Claw energy, for when you need the focus of a hawk and the energy of a cheetah. A warning label says drinking too much may cause heart damage, liver damage, and not living anymore. \
 	Red cans are drunk faster."
-	icon_state = "claw_blue"
+	icon_state = "claw_red"
 	center_of_mass = list("x"=16, "y"=6)
 	preloaded_reagents = list("energy_drink_monster" = 25, "sodawater" = 25, "cherryjelly" = 10)
 	volume = 60
 
 /obj/item/reagent_containers/food/drinks/cans/monster_blue
 	name = "\"Code BLUE\" Claw Energy"
-	desc = "Claw energy, for when you need the focus of a hawk and the energy of a cheetah. A warning label says drinking to much may cause heart damage, liver damage, and not living anymore. \
+	desc = "Claw energy, for when you need the focus of a hawk and the energy of a cheetah. A warning label says drinking too much may cause heart damage, liver damage, and not living anymore. \
 	Code BLUE being counted as a mellow smooth taste that would be best cold."
-	icon_state = "claw_red"
+	icon_state = "claw_blue"
```

Comment:
```suggestion
	icon_state = "claw_red"
```

Reply:
Trilby please, no memeing, look at the icons, look at the names, like the cigarettes they have their respective sprites